scripts/controlnet.py

```diff
@@ -355,10 +355,10 @@ def refresh_all_models(*inputs):
         if shared.opts.data.get("controlnet_disable_control_type", False):
             type_filter = None
         else:
-            with gr.Row():
-                type_filter = gr.Radio(list(preprocessor_filters.keys()), label=f"Control Type", value='All', elem_id=f'{elem_id_tabname}_{tabname}_controlnet_type_filter_radio')
+            with gr.Row(elem_id="controlnet_control_type"):
```

Comment:
I think use `elem_id` here will cause element id conflict among multiple controlnet ui-groups. Please replace `elem_id` with `elem_classes`. Same in line 361.

Reply:
Okay, classes it is.